resources/web/server/simulation_server.py

```diff
@@ -249,7 +249,10 @@ def runWebotsInThread(client):
             global config
             world = self.project_instance_path + '/worlds/' + self.world
             port = client.streaming_server_port
-            command = config['webots'] + ' --batch --mode=pause --minimize '
+            command = config['webots'] + ' --batch --mode=pause '
+            # on Windows, the MJPEG stream won't work if the Webots window is minimized
```

Comment:
in my previous suggestion I forgot to remove the Windows special case from the comment
```suggestion
            # the MJPEG stream won't work if the Webots window is minimized
```